c/test/check_sbp.c

```diff
@@ -83,11 +83,21 @@ u8 last_msg[256];
 void* last_context;
 
 
+u32 n_frame_callbacks_logged;
+u16 last_frame_sender_id;
+u16 last_frame_msg_type;
+u8 last_frame_payload_len;
+u16 last_frame_len;
+void* last_frame_context;
+u8 last_frame[256 + 8];
+
 void logging_reset(void)
 {
   n_callbacks_logged = 0;
+  n_frame_callbacks_logged = 0;
```

Comment:
should the `last_*` variables get reset too?

Reply:
yes, that would be better. updated.